core/templates/dev/head/pages/landing/LandingPage.js

```diff
@@ -0,0 +1,167 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for landing page.
+ */
+
+oppia.constant('LANDING_PAGE_DATA', {
+  maths: {
+    fractions: {
```

Comment:
How about ratios?

Reply:
@seanlip, I didn't find enough data (images) to create ratios landing page. (I have two images with me and 6 are needed)

Reply:
Ah -- actually, I forgot one thing. Drop the "student" landing page for Ratios, we'll just have teachers only. (If you need to we can drop it for Fractions as well.) We found that the "teacher" one was more important.